src/participant_network.star

```diff
@@ -165,6 +165,24 @@ def launch_participant_network(
         final_genesis_timestamp = constants.GENESIS_TIME[network_params.network]
         network_id = constants.NETWORK_ID[network_params.network]
         validator_data = None
+    elif network_params.network == "ephemery":
+        el_cl_genesis_data_uuid = plan.run_sh(
```

Comment:
can we take this out into a function; might make things easier to read

Reply:
https://github.com/kurtosis-tech/ethereum-package/pull/459/commits/36a151c0e1a639c45873c0194f9333243330ccbc does this make you happy enough? 

Don't think it needs to be a whole different function tbh. Its very small script.

Reply:
i was wondering if the entire conditonal could be converted into a function, but i see the `kurtosis` block is massive too. Merge it now, lets revisit later.

Reply:
Resolving this!